.github/workflows/prepare-for-minor-release.yml

```
@@ -56,90 +78,87 @@ jobs:
         with:
           token: ${{ secrets.REPO_DISPATCH_TOKEN }}
           repository: ${{ matrix.repo }}
-          event-type: az_quickstart_prep_for_minor_release
+          event-type: az_quickstart_prep_for_${{ inputs.release_type }}_release
           client-payload: '{"release_branch_name": "${{ inputs.release_branch_name }}" , "this_release_branch_alias_constraint": "${{needs.branch_name_prep.outputs.this_release_branch_alias_constraint}}", "main_branch_alias": "${{needs.branch_name_prep.outputs.main_branch_alias}}", "next_release_branch_alias_constraint": "${{needs.branch_name_prep.outputs.next_release_branch_alias_constraint}}"}'
```

Would it be better to always dispatch the same event type but include an additional variable in the payload?
```suggestion
          event-type: az_quickstart_prep_for_new_release
          client-payload: '{"release_branch_name": "${{ inputs.release_branch_name }}",  "release_type": "${{ inputs.release_type }}", "this_release_branch_alias_constraint": "${{needs.branch_name_prep.outputs.this_release_branch_alias_constraint}}", "main_branch_alias": "${{needs.branch_name_prep.outputs.main_branch_alias}}", "next_release_branch_alias_constraint": "${{needs.branch_name_prep.outputs.next_release_branch_alias_constraint}}"}'
```